docker-for-mac/faqs.md

```diff
@@ -184,7 +184,43 @@ For full information on adding server and client side certs, see [Adding
 TLS certificates](/docker-for-mac/index.md#adding-tls-certificates) in
 the Getting Started topic.
 
-### How do I reduce the size of Docker.qcow2?
+### Disk Usage
+
+#### Qcow2 or Raw?
+
+By default Docker for Mac stores containers and images in a file
+`Docker.raw` or `Docker.qcow2` in the directory
+`~/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux`.
+Starting with High Sierra with APFS (Apple Filesystem) enabled, Docker
+uses the "raw" format (`Docker.raw`), otherwise it uses the Qcow2
+format (`Docker.qcow2`).
+
+#### `Docker.raw` consumes an insane amount of disk space!
+
+This is an illusion.  Docker uses the raw format on Macs running the
+Apple Filesystem (APFS).  APFS supports sparse files, i.e., files
```

Comment:
reword to avoid the "i.e.": APFS supports _sparse files_, which compress long runs of zeroes representing unused space. The output of `ls` is misleading, because it lists the logical size of the file rather than its physical size. To see the physical size, add the `-s` switch: